pageserver/src/tenant/storage_layer/layer.rs

```diff
@@ -259,7 +259,7 @@ impl Layer {
 
         layer
             .get_value_reconstruct_data(key, lsn_range, reconstruct_data, &self.0, ctx)
-            .instrument(tracing::info_span!("get_value_reconstruct_data", layer=%self))
+            .instrument(tracing::debug_span!("get_value_reconstruct_data", layer=%self))
```

Comment:
Downside of this is that permanent layer opening errors now will not have a this layer information. Perhaps we just remove this span and add the logging to line 1268:

```patch
-                    tracing::error!("layer loading failed permanently: {permanent:#}");
+                    tracing::error!("layer loading failed permanently for {owner}: {permanent:#}");
```

together with:

```suggestion
```

Reply:
Thanks, committed.